website/docs/d/signalr_web_pubsub.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
since we're reverting the `signalr_` prefix, we should be able to revert the docs here?